proposals/lip-0057.md

```diff
@@ -113,11 +113,11 @@ For the rest of this proposal we define the following constants:
 | `MAX_NUMBER_STAKING_SLOTS` | uint32 | `10` | The maximum size of the stakes array of a staker substore entry. |
 | `MAX_NUMBER_PENDING_UNLOCKS` | uint32 | `20` | The maximum size of the pendingUnlocks array of a staker substore entry. |
 | `FAIL_SAFE_MISSED_BLOCKS` | uint32 | `50` | The number of consecutive missed blocks used in the fail safe banning mechanism. |
-| `FAIL_SAFE_INACTIVE_WINDOW` | uint32 | `130,000` | The length of the inactivity window used in the fail safe banning mechanism. |
-| `LOCKING_PERIOD_STAKING` | uint32 | `26,000` | The [locking period][lip-0023#explicit-unlock-mechanism] for regular staked tokens. |
-| `LOCKING_PERIOD_SELF_STAKING` | uint32 | `260,000` | The [locking period][lip-0023#explicit-unlock-mechanism] for self-staked tokens. |
-| `PUNISHMENT_WINDOW_STAKING` | uint32 | `260,000` | The punishment period for regular stakes, i.e., how many blocks after misbehavior report stakers can unlock tokens unstaked from a punished validator. |
-| `PUNISHMENT_WINDOW_SELF_STAKING` | uint32 | `780,000` | The punishment period for self-stakes, i.e., how many blocks after misbehavior report punished validators can unlock their self-unstaked tokens. |
+| `FAIL_SAFE_INACTIVE_WINDOW` | uint32 | 120960 = 14x24x360 | The length of the inactivity window used in the fail safe banning mechanism, measured in number of blocks. |
+| `LOCKING_PERIOD_STAKING` | uint32 | 25,920 = 3x24x360 | The [locking period][lip-0023#explicit-unlock-mechanism] for regular staked tokens, measured in number of blocks. |
+| `LOCKING_PERIOD_SELF_STAKING` | uint32 | 241,920 = 28x24x360 | The [locking period][lip-0023#explicit-unlock-mechanism] for self-staked tokens. |
+| `PUNISHMENT_WINDOW_STAKING` | uint32 | 241,920 = 28x24x360 | The punishment period for regular stakes, i.e., how many blocks after misbehavior report stakers can unlock tokens unstaked from a punished validator. |
+| `PUNISHMENT_WINDOW_SELF_STAKING` | uint32 | 786,240 = 13x7x24x3600 | The punishment period for self-stakes, i.e., how many blocks after misbehavior report punished validators can unlock their self-unstaked tokens. |
```

Comment:
Maybe 3 * LOCKING_PERIOD_SELF_STAKING = ... That we now compute this constant using weeks and the rounding up to 13 weeks seems also quite arbitrarily.

Reply:
I wrote in the issue description that we use weeks instead of months as a unit of measure.

About the actual value, ok, makes more sense to just multiply by 3.